Python/tier2_redundancy_eliminator_bytecodes.c

```diff
@@ -189,6 +177,58 @@ dummy_func(void) {
         }
     }
 
+    op(_TO_BOOL, (value -- res)) {
+        (void)value;
+        OUT_OF_SPACE_IF_NULL(res = sym_new_known_type(ctx, &PyBool_Type));
```

Comment:
Honestly I'd prefer to break code like this up in two lines:
```suggestion
        res = sym_new_known_type(ctx, &PyBool_Type);
        OUT_OF_SPACE_IF_NULL(res);
```
I think the expression is "don't bury the lede". :-)